docs/releasing/publishing.md

```diff
@@ -48,32 +64,36 @@
   npm unlink ../govuk-frontend/package/
   ```
 
-11. Create a pull request and copy the changelog text.
+10. Create a pull request and copy the changelog text.
    When reviewing the PR, check that the version numbers have been updated and that the compiled assets use this version number.
 
-12. Once the pull request is approved, merge to **master**.
+11. Once the pull request is approved, merge to **master**.
 
-13. Checkout **master** and pull the latest changes.
+12. Checkout **master** and pull the latest changes.
 
-14. Log into npm (`npm login`), using team [credentials](https://github.com/alphagov/design-system-team-credentials/tree/master/npm/govuk-patterns-and-tools).
+13. Log into npm (`npm login`), using team [credentials](https://github.com/alphagov/design-system-team-credentials/tree/master/npm/govuk-patterns-and-tools).
 
-15. Run `npm run publish-release`, you will be prompted to continue or cancel.
+14. Run `npm run publish-release`, you will be prompted to continue or cancel.
 
-16. View the created tag in the [Github interface](https://github.com/alphagov/govuk-frontend/releases)
+15. View the created tag in the [Github interface](https://github.com/alphagov/govuk-frontend/releases)
   - select the latest tag
   - press 'Edit tag'
   - set "GOV.UK Frontend v[version-number]" as the title
   - add release notes from changelog
   - attach the generated ZIP that has been generated at the root of this project
   - publish release
 
-17. Log out from npm
+16. Log out from npm
 ```bash
 npm logout
 ```
 
-18. Once you've updated the GOV.UK Design System, Prototype Kit, and Frontend Docs, post a short summary of the release in the cross-government and GDS #govuk-design-system Slack channels. For example:
+# Steps you must complete after you release a new version of GOV.UK Frontend
+
+1. Update the GOV.UK Design System, Prototype Kit, and Frontend Docs to use a new release of GOV.UK Frontend, and release a new version of the Prototype Kit.
+
+2. Post your summary of the release in the [cross-government](https://ukgovernmentdigital.slack.com/archives/C6DMEH5R6) and [GDS #govuk-design-system](https://gds.slack.com/archives/CAF8JA25U) Slack channels.
```

Comment:
If we're mentioning the separate release notes for the prototype kit in the 'Before' steps, could we split this out to say something like: 

"Post your summary of the GOV.UK Frontend release in the govuk-design-system Slack channels within GDS and x-gov slack."
"Post your summary of the Prototype kit release in the prototype-kit Slack channel on x-gov slack"

Reply:
```suggestion
2. Post your summary of the:
- GOV.UK Frontend release in the govuk-design-system channels on [GDS Slack](https://gds.slack.com/archives/CAF8JA25U) and [x-gov Slack](https://ukgovernmentdigital.slack.com/archives/C6DMEH5R6)
- Prototype kit release in the [prototype-kit channel](https://ukgovernmentdigital.slack.com/archives/C0647LW4R) on x-gov Slack
```